app/views/metadata_fields/_licences.html.erb

```diff
@@ -28,11 +28,17 @@
   %>
 <% end %>
 
-<%= render layout: "shared/form_group", locals: { f: f, field: :licence_transaction_will_continue_on, label: "Name of website where you apply" } do %>
+<div class="additional-field-context-section">
+  <h2>How users apply</h3>
```

Comment:
Small thing, the browser will likely fix this anyway when rendering the page, but the opening tag here is a `h2`, and the closing tag is a `h3`.

Reply:
Good spot, now fixed!